src/core_plugins/input_control_vis/public/vis_controller.js

```diff
@@ -7,14 +7,22 @@ class VisController {
   constructor(el, vis) {
     this.el = el;
     this.vis = vis;
+    this.lastTimeFilter = JSON.stringify(this.vis.API.timeFilter.time);
```

Comment:
Instead of using `JSON.stringify` to determine if the time filter's changed, can we use `_.clone` and `_.isEqual` instead?